picard/mbxml.py

```diff
@@ -195,7 +195,7 @@ def artist_credit_to_metadata(node, m, release=False):
         m["musicbrainz_albumartistid"] = ids
         m["albumartist"] = artist
         m["albumartistsort"] = artistsort
-        m["~albumartists"] = artists
```

Comment:
the '~' simple means its a 'hidden/protected' tag and is appeneded to hide it from UI/saving from file tags.